advanced/am-pattern-2/templates/am-analytics/worker-2/wso2am-pattern-2-am-analytics-worker-conf.yaml

```diff
@@ -28,7 +28,7 @@ data:
   {{- else }}
```

Comment:
@Chathuru let us rename the ConfigMap as highlighted earlier.

Reply:
Fix in https://github.com/wso2/kubernetes-apim/pull/365/commits/656646538e8cc1b8d8fbc51cb3d206a1c58101e5